server/src/infra/infra.utils.ts

```diff
@@ -180,6 +193,18 @@ export function searchAssetBuilder(
     builder.leftJoinAndSelect(`${builder.alias}.smartInfo`, 'smartInfo');
   }
 
+  if (options.personIds && options.personIds.length > 0) {
+    builder
+      .leftJoin(`${builder.alias}.faces`, 'faces')
+      .andWhere('faces.personId IN (:...personIds)', { personIds: options.personIds })
+      .addGroupBy(`${builder.alias}.id`)
+      .having('COUNT(faces.id) = :personCount', { personCount: options.personIds.length });
```

Comment:
I don't understand this. Why are we checking that the face count is equal to the person count?

Reply:
Let say you select people A and people B. Without that condition, the result will include people A only or people B only. We want the result includes both people A and people B altogether